CMakeLists.txt

```diff
@@ -65,12 +65,13 @@ target_include_directories(pmtv INTERFACE ${pmt_SOURCE_DIR}/include/)
 
 # FFTW3 is build 2 times for float and double precisions
 SET(FFTW_PREFIX ${CMAKE_CURRENT_BINARY_DIR}/fftw)
+SET(FFTW_CFLAGS "${CFLAGS} -w")
 if (EMSCRIPTEN)
     SET(FFTW_CONFIG cd ${FFTW_PREFIX}/src/ && emconfigure ./configure --quiet --disable-fortran --prefix=${FFTW_PREFIX}/install)
-    SET(FFTW_BUILD emmake make -j && emmake make install && emmake make clean)
+    SET(FFTW_BUILD emmake make -j CFLAGS=${FFTW_CFLAGS} && emmake make install && emmake make clean)
```

Comment:
Looking at the build output I think we might also want to pass `--silent` to make to suppress the echo of the compile commands it is running? Right now the CI build step mostly shows the output of make for fftw (and that even two times).

Reply:
I included `--silent`. There is still some output form `fftw`, I'm not sure if we can control it.